components/elements/button/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { forwardRef } from "react";
+interface ButtonProps extends React.ComponentPropsWithoutRef<"button"> {
+  children: React.ReactNode;
+}
+
+const Button = forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
why not `const Button: React.FC = (`?

Reply:
I did have the React.FC but I removed it in favour of the [docs recommendation](https://github.com/typescript-cheatsheets/react#function-components). Further down there is a section on why React.FC is now discouraged.